aad/azure-spring-boot-starter-active-directory-b2c/aad-b2c-resource-server/pom.xml

```diff
@@ -4,9 +4,10 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.5.0</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-parent;external_dependency} -->
+    <groupId>com.azure.spring</groupId>
+    <artifactId>azure-spring-boot-samples</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../../pom.xml</relativePath>
```

Comment:
Use `azure-spring-boot-samples` as parent is convenient for us, but it's not convenient for our customer, so my suggestion is to import pom in each sample instead of using parent pom.

Reply:
that depends how we see the samples here.
If we hope our customer can copy and paste code easily, if they wnat to create their own project, I agree with you, but from the maintainer side, if we focus on to let our customer run samples and to know the basic funtions of our libraries, this way is better for us to maintaine.